From 51043fb1ad5b99a66749bce2661b388e26b6f90f Mon Sep 17 00:00:00 2001
From: michpappas <mpappas@fastmail.fm>
Date: Sat, 21 Jan 2023 18:11:26 +0100
Subject: [PATCH] Disable all user options to allow controlling via KConfig

---
 code/gcov_public.h | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/code/gcov_public.h b/code/gcov_public.h
index 0982c2c..9a32a8c 100644
--- a/code/gcov_public.h
+++ b/code/gcov_public.h
@@ -51,7 +51,7 @@
  * If defined, you must also provide defs below
  * for GCOV_PRINT_STR and GCOV_PRINT_NUM.
  */
-#define GCOV_OPT_PRINT_STATUS
+//#define GCOV_OPT_PRINT_STATUS
 
 /* Reset watchdog timeout during gcov tree scanning.
  * Might be needed if you enable serial output on a slow port,
@@ -95,7 +95,7 @@ extern void *__ctor_end;
  * You might NOT want this if you are extremely memory constrained
  * (such as in PROM) and do not need this function to take up some bytes.
  */
-#define GCOV_OPT_PROVIDE_CLEAR_COUNTERS
+//#define GCOV_OPT_PROVIDE_CLEAR_COUNTERS
 
 /* Provide small imitation printf function.
  * This is only needed if you want serial port outputs and
@@ -104,7 +104,7 @@ extern void *__ctor_end;
  * write_bytes() that does the actual serial output in your system.
  * See gcov_printf.c
  */
-#define GCOV_OPT_PROVIDE_PRINTF_IMITATION
+//#define GCOV_OPT_PROVIDE_PRINTF_IMITATION
 
 /* select data output method(s) ------------------------------------ */
 
@@ -122,7 +122,7 @@ extern void *__ctor_end;
 
 /* Modify this output filename if desired */
 /* Not used if you do not define GCOV_OPT_OUTPUT_BINARY_FILE */
-#define GCOV_OUTPUT_BINARY_FILENAME "gcov_output.bin"
+//#define GCOV_OUTPUT_BINARY_FILENAME "gcov_output.bin"
 
 /* Modify file headers, data type and functions, if needed */
 /* Not used if you do not define GCOV_OPT_OUTPUT_BINARY_FILE */
@@ -164,7 +164,7 @@ typedef FILE * GCOV_FILE_TYPE;
  * for GCOV_PRINT_STR and GCOV_PRINT_NUM.
  * Can be combined with other GCOV_OPT_OUTPUT_* options.
  */
-#define GCOV_OPT_OUTPUT_SERIAL_HEXDUMP
+//#define GCOV_OPT_OUTPUT_SERIAL_HEXDUMP
 
 /* Function to print a string without newline.
  * Not used if you don't define either GCOV_OPT_PRINT_STATUS
-- 
2.25.1

